src/DtoTransformer/DtoTransformer/DtoGenerator.cs

```diff
@@ -0,0 +1,105 @@
+﻿using VDS.RDF;
+using VDS.RDF.Query;
+using Graph = VDS.RDF.Graph;
+
+public class DtoGenerator
+{
+    public static ReviewDTO GenerateDto(Graph graph)
+    {
+        // Run first sparql query retrieving Review data
+        string query = @"
+    PREFIX review: <https://rdf.equinor.com/ontology/review/>
+    PREFIX prov: <http://www.w3.org/ns/prov#>
+    PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+    prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+    prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+    prefix exdata: <https://example.com/data/>
+    prefix exdoc: <https://example.com/doc/>
+    prefix rec: <https://rdf.equinor.com/ontology/record/>
+    prefix rev: <https://rdf.equinor.com/ontology/revision/>
+    prefix rdl: <http://example.com/rdl/>
+    prefix mel: <https://rdf.equinor.com/ontology/mel/v1#>
+    SELECT ?reviewId ?aboutVersion ?issuedBy ?generatedAtTime ?reviewStatus ?label
+    WHERE {
+        ?reviewId a review:Review ;
+                  review:aboutVersion ?aboutVersion ;
+                  review:issuedBy ?issuedBy ;
+                  prov:generatedAtTime ?generatedAtTime ;
+                  rdf:type ?reviewStatus ;
+                  rdfs:label ?label .
+        FILTER (?reviewStatus != review:Review)
+    }";
+
+        ReviewDTO reviewDto = new ReviewDTO();
+
+        SparqlResultSet results = (SparqlResultSet)graph.ExecuteQuery(query);
+        if (results.Count > 0)
+        {
+            SparqlResult result = (SparqlResult)results[0];
+            reviewDto.ReviewId = result["reviewId"].ToString();
+            reviewDto.AboutVersion = result["aboutVersion"].ToString();
+            reviewDto.IssuedBy = result["issuedBy"].ToString().Split("^^")[0];
+            reviewDto.GeneratedAtTime = DateOnly.Parse(result["generatedAtTime"].ToString().Split("^^")[0]);
+            reviewDto.ReviewStatus = result["reviewStatus"].ToString().Split("^^")[0];
```

Comment:
```suggestion
            reviewDto.IssuedBy = ((LiteralNode)result["issuedBy"]).Value;
```
Better way to get the value from a literal (granted you know for sure it is a literal node)